server/instance_cloud_oauth.go

```diff
@@ -98,12 +98,14 @@ func (ci *cloudOAuthInstance) GetClient(connection *Connection) (Client, error)
 func (ci *cloudOAuthInstance) getClientForConnection(connection *Connection) (*jira.Client, *http.Client, error) {
 	oauth2Conf := ci.GetOAuthConfig()
 	ctx := context.Background()
-	tokenSource := oauth2Conf.TokenSource(ctx, connection.OAuth2Token)
-	if ci.JWTInstance != nil && tokenSource == nil {
-		ci.Plugin.API.LogDebug("Returning a JWT token client in case the stored JWT instance is not nil and the user's oauth token is nil")
+
+	// Checking if this user's connection is for a JWT instance
+	if ci.JWTInstance != nil && connection.OAuth2Token == nil {
+		ci.Plugin.API.LogDebug("Returning a JWT token client since the stored JWT instance is not nil and the user's oauth token is nil")
```

Comment:
Are you thinking we should remove log statements like this in a following release?

Reply:
@mickmister Yes, we will remove these in a later release. These are just to check if the code executed as we expected it to.